lib/jarvis/commands/merge.rb

```diff
@@ -0,0 +1,15 @@
+require "clamp"
+
+module Jarvis module Command class Merge < Clamp::Command
+
+  banner "Merge a pull request into one or more branches."
+
+  parameter "URL", "The URL to merge"
+  parameter "BRANCHES ...", "The branches to merge"
+
+  def execute
+    puts "Merging: #{url} into #{branches_list}"
```

Comment:
Mock implementation; mostly as a  way for me to explore Clamp::Command parameters and options via Lita.